sdks/java/io/hadoop-input-format/jdk1.8-hifio-tests/src/test/resources/cassandra.yaml

```diff
@@ -0,0 +1,1074 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
this file is named .yaml, but the other yaml files in this check in are named .yml. Is there a reason for the difference?